<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgNewDistrict</class>
 <widget class="QDialog" name="dlgNewDistrict">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New District</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblDistrictNo">
     <property name="statusTip">
      <string>Number of the district</string>
     </property>
     <property name="text">
      <string>District Number</string>
     </property>
     <property name="buddy">
      <cstring>sbxDistrictNo</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblMembers">
     <property name="text">
      <string>Members</string>
     </property>
     <property name="buddy">
      <cstring>sbxMembers</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblDescription">
     <property name="text">
      <string>Description</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="buddy">
      <cstring>txtDescription</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPlainTextEdit" name="txtDescription">
     <property name="toolTip">
      <string>Description of the district</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="sbxMembers">
     <property name="toolTip">
      <string>Number of members elected from the district</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>District Name</string>
     </property>
     <property name="buddy">
      <cstring>inpName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="inpName">
     <property name="toolTip">
      <string>Name of the district</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="RdsDistrictSpinBox" name="sbxDistrictNo">
     <property name="toolTip">
      <string>Number of the district</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RdsDistrictSpinBox</class>
   <extends>QSpinBox</extends>
   <header>.RedistrictingWidgets</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sbxDistrictNo</tabstop>
  <tabstop>inpName</tabstop>
  <tabstop>sbxMembers</tabstop>
  <tabstop>txtDescription</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgNewDistrict</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgNewDistrict</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
